src/project/init.rs

```diff
@@ -188,6 +188,9 @@ fn ask_name(methods: &Methods, dir: &Path, options: &Init)
             if confirm.ask()? {
                 return Ok((target_name, true));
             }
+        } else if is_local_dev(&target_name) {
+            eprintln!("no remote instance _localdev found, \
```

Comment:
I think it's more error of a kind `_localdev instance name is reserved. You can't initialize instance with this name, but you can link existing instance`. Because otherwise it's not clear why you have `to link first`. And we don't want users to link `_localdev` we want them to run `edb server`.